front/style/pages/_dashboard.scss

```diff
@@ -0,0 +1,97 @@
+.dashboard {
+    margin-bottom: 25px;
+    a {
+        text-decoration: none;
+    }
+
+    .newblue {
+        color: $blue--dark;
+    }
+
+    .button {
+        width: 199px;
+        height: 47px;
+        border: solid 1px #000000;
+        background-color: #ffffff;
+        padding: 10px;
+        margin: 10px 0;
+        text-transform: uppercase;
+    }
+
+    .dashboard_tag {
+        height: 24px;
+        border-radius: 12px;
+        background-color: #ebebeb;
```

Comment:
You can use maybe the `$gray` here ?

You can check if similar chart colors are already defined here : https://github.com/EnMarche/en-marche.fr/blob/master/front/style/mixins/_colors.scss

Reply:
thanks, done